packages/kit/src/send-and-confirm-durable-nonce-transaction.ts

```diff
@@ -18,10 +18,45 @@ type SendAndConfirmDurableNonceTransactionFunction = (
 ) => Promise<void>;
 
 type SendAndConfirmDurableNonceTransactionFactoryConfig<TCluster> = {
+    /** An object that supports the {@link GetSignatureStatusesApi} and the {@link SendTransactionApi} of the Solana RPC API */
     rpc: Rpc<GetAccountInfoApi & GetSignatureStatusesApi & SendTransactionApi> & { '~cluster'?: TCluster };
+    /** An object that supports the {@link AccountNotificationsApi} and the {@link SignatureNotificationsApi} of the Solana RPC Subscriptions API */
     rpcSubscriptions: RpcSubscriptions<AccountNotificationsApi & SignatureNotificationsApi> & { '~cluster'?: TCluster };
 };
 
+/**
+ * Returns a function that you can call to send a durable-nonce-based transaction to the network and
+ * to wait until it has been confirmed.
+ *
+ * @param config
```

Comment:
These just make the param appear as `config` in the docs instead of `__namedParameters`.